src/zoom_type.h

```diff
@@ -46,14 +43,14 @@ enum ZoomLevel {
 
 	ZOOM_LVL_MIN      = ZOOM_LVL_NORMAL, ///< Minimum zoom level.
 	ZOOM_LVL_MAX      = ZOOM_LVL_OUT_32X, ///< Maximum zoom level.
+
+	ZOOM_LVL_SHIFT    = 2,
+	ZOOM_LVL_BASE     = 1 << ZOOM_LVL_SHIFT,
```

Comment:
Not sure ZOOM_LVL_SHIFT should be a ZoomLevel member.

Reply:
What else should it be? I moved it (both of them) in because of differing signedness when adding/whatever, which didn't seem necessary

Reply:
Just a (u)int as it was seems more correct to me.